client/my-sites/checkout/composite-checkout/components/wp-checkout-order-summary.js

```diff
@@ -279,7 +279,7 @@ function CheckoutSummaryPlanFeatures( { isMonthlyPricingTest } ) {
 
 	return (
 		<>
-			{ planFeatures.filter( Boolean ).map( ( feature ) => {
+			{ planFeatures.map( ( feature ) => {
 				const isSupported = ! feature.startsWith( '~~' );
```

Comment:
```suggestion
				const isSupported = feature.indexOf( '~~' ) !== 0;
```

According to the error message you mentioned `e.startsWith is not a function. (In 'e.startsWith(\"~~\")', 'e.startsWith' is undefined)`, the problem occurs because the browser doesn't support `startsWith` method for strings. So we should replace this line with the suggestion above. The `.filter( Boolean )` would be okay as is.

Reply:
I disagree. Can we try this fix as it is and then see if the errors continue before making the change you suggested? (If we can or use startsWith then many other places in calypso need to changed.)

Reply:
I looked into these errors and they happened on recent versions of Chrome and Safari. This was my first instinct as well, but I think `startsWith` should be present on strings.